### PR TITLE
havesexwith.me

### PR DESCRIPTION
**Pull request by breixopd**
image
image

### CODE REVIEW

File: websites/H/havesexwith.me/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    "webmail.havesexwith.me",
+    "does-only.fans",
+    "rides-pen.is",
+    "cocks-and.rocks",
+    "faps-to.men",
+    "getscammed.lol"
+  ],
+  "description": {
+    "en": "Extension for havesexwith.me sites and tools"
+  },
```
Comment (Bas950) [inline websites/H/havesexwith.me/dist/metadata.json:22]: should be the description of the wesbite

File: websites/H/havesexwith.me/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  "service": "havesexwith.me",
+  "version": "1.0.0",
+  "logo": "https://imgur.com/a/6bSfmtP",
+  "thumbnail": "https://imgur.com/a/TBoJZxW",
```
Comment (Bas950) [inline websites/H/havesexwith.me/dist/metadata.json:26]: should be direct image links IE ending with .png / ,jpg

File: websites/H/havesexwith.me/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  "color": "#FF0000",
+  "tags": [
+    "site",
+    "fun",
+    "funny",
+    "website"
+  ],
+  "category": "nsfw"
```
Comment (Bas950) [inline websites/H/havesexwith.me/dist/metadata.json:34]: nsfw is a tag, not a category see docs

File: websites/H/havesexwith.me/presence.ts [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+	
+    if (presenceData.details == null) {
+        //This will fire if you do not set presence details
+        presence.setTrayTitle(); //Clears the tray title for mac users
+        presence.setActivity(); /*Update the presence with no data, therefore clearing it and making the large image the Discord Application icon, and the text the Discord Application name*/
+    } else {
+        //This will fire if you set presence details
+        presence.setActivity(presenceData); //Update the presence with all the values from the presenceData object
+    }
+});
```
Comment (Bas950) [inline websites/H/havesexwith.me/presence.ts:34]: no var
remove unneeded comments

and this is not enough that get your presence on our website,
there must be some variation in the statuses.

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/H/havesexwith.me/presence.ts (hunk 4 above)
Comment (breixopd) [reply websites/H/havesexwith.me/presence.ts]: Im adding a random comment every time you change url, also, what does Codacy want? Its asking me to change the variables to constants even though I used the file in https://docs.premid.app/en/dev/presence to start

**Comment by breixopd** [comment]
Is it fine now?

File: websites/H/havesexwith.me/presence.ts [hunk 5]
```diff
@@ -0,0 +1,29 @@
+const state = ["Being epic", "Just browsing", "Chilling", "You should check this out", "Now this, is epic"][Math.floor(Math.random() * 3)];
+const smolimage = ["You should cop an email", "Hello there!", "Smol doggo"][Math.floor(Math.random() * 3)];
```
Comment (Bas950) [inline websites/H/havesexwith.me/presence.ts:2]: yea PreMiD isn't for you to advertise, you need to add informational statuses stating what you are doing/ where you are on the page.
Else your presence will be denied.

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Comment by ririxidev** [comment]
Okey so, site & presence is low quality and we are not going to accept this like our guidelines says:

**Pull request closed without merging**